core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/keywordSearch/KeywordSearchOpExec.scala

```diff
@@ -3,12 +3,12 @@ package edu.uci.ics.texera.workflow.operators.keywordSearch
 import edu.uci.ics.amber.engine.common.model.tuple.Tuple
 import edu.uci.ics.texera.workflow.common.operators.filter.FilterOpExec
 import org.apache.lucene.queryparser.classic.QueryParser
-import org.apache.lucene.analysis.core.SimpleAnalyzer
+import org.apache.lucene.analysis.standard.StandardAnalyzer
 import org.apache.lucene.index.memory.MemoryIndex
 import org.apache.lucene.search.Query
 
 class KeywordSearchOpExec(attributeName: String, keyword: String) extends FilterOpExec {
-  @transient private lazy val analyzer = new SimpleAnalyzer()
+  @transient private lazy val analyzer = new StandardAnalyzer()
```

Comment:
please add a comment here to briefly explain why we choose StandardAnalyzer.

Reply:
Addressed!